src/Microsoft.DocAsCode.MarkdigEngine.Extensions/QuoteSectionNote/QuoteSectionNoteRender.cs

```diff
@@ -88,8 +88,12 @@ private void WriteVideo(HtmlRenderer renderer, QuoteSectionNoteBlock obj)
             renderer.WriteLine("</div>");
         }
 
-        private static string FixUpLink(string link)
+        public static string FixUpLink(string link)
         {
+            if (!link.Contains("https"))
```

Comment:
Consider using `UriBuilder` to construct an https link in line 124 `{ Schema = "https" }` ?